_data/simple-icons.json

```diff
@@ -9407,7 +9407,12 @@
         },
         {
             "title": "Toggl",
-            "hex": "E01B22",
+            "hex": "FFDE91",
+            "source": "https://toggl.com/media-toolkit"
+        },
+        {
+            "title": "Toggl Track",
+            "hex": "E57CD8",
             "source": "https://toggl.com/media-toolkit"
```

Comment:
Did not review the icons, but I think the link should be https://toggl.com/track/media-toolkit/ since the current one is redirecting there.

Reply:
done

Reply:
I was more talking about the `/track` part of the URL

Reply:
done